pkg/southbound/gnmi/conn.go

```diff
@@ -69,14 +69,15 @@ func newDestination(target *topoapi.Object) (*baseClient.Destination, error) {
 	}
 
 	if tlsOptions.Plain {
-		log.Info("Plain (non TLS) connection to ", configurable.Address)
+		log.Infof("Plain (non TLS) connection to %s", configurable.Address)
 	} else {
+		log.Infof("TLS connection to %s", configurable.Address)
 		tlsConfig := &tls.Config{}
 		if tlsOptions.Insecure {
-			log.Info("Insecure TLS connection to ", configurable.Address)
+			log.Infof("Skipping verification of certificate chain for target %s", configurable.Address)
```

Comment:
I think you should mention TLS somewhere in the message

Reply:
Done